src/components/Editor/index.js

```diff
@@ -92,15 +92,6 @@ class Editor extends PureComponent {
       highlightedLineRange: null,
       editor: null
     };
```

Comment:
We can also use the class properties for state, and all of the other vars set in this constructor

Reply:
(and then this overridden constructor can be removed)